### PR TITLE
Minor updates to QS overview markdown

### PR DESCRIPTION
**Pull request by case-eee**
@tmikeschu @laurenfazah here are some minor updates to the QS markdown. some of them were typos, some minor organization, and some added additional detail for the project.

to do: create the Rails API starter repo (unless we feel like that's another thing they could do over intermission week?)

### CODE REVIEW

**Comment by laurenfazah** [comment]
@case-eee looks good - should we keep the PR open while it's a WIP?

File: module4/projects/quantified-self/quantified-self.md [hunk 1]
```diff
@@ -65,50 +55,61 @@ Your requirements for the application are detailed in the cards you're going to
...
 - All elements from the mockup should be represented in your app, even if the styling or layout is different.  Unless it is mentioned specifically in the user stories, colors and other styling are up to you. You'll be just fine if it looks exactly like the mockup though.
-- Details, Details, Details!!! If it's in the user stories, we're looking for it in your app. And ask before you add any additional functionality. The job is to build what is asked for, and get any improvements approved first.
-- This is a 2 week project without any scheduled instructor check ins. We expect that you'll be tagging instructors in PRs on github wherever you'd like feedback. We also expect that if you feel like your team is falling behind, that you'll reach out for extra support.
+- Details, Details, Details!!! If it's in the user stories, we're looking for it in your app. Please ask before you add any additional functionality. The job is to build what is asked for, and get any improvements approved first.
```
Comment (tmikeschu) [inline module4/projects/quantified-self/quantified-self.md]: I feel like this one should have bold text haha

File: module4/projects/quantified-self/quantified-self.md [hunk 2]
```diff
@@ -65,50 +55,61 @@ Your requirements for the application are detailed in the cards you're going to
...
 - EXTENSION: Build an admin panel to manage data
 - EXTENSION: Add offline functionality
 
 #### Broad Back End features
 
-- Build a service in Node that will store all of the Quantified Self data
-- The service will serve and consume JSON
-- Use SQL for all database communication
-- Document a plan in the form of a schema, API docs and user stories.
+First, we'll have our Front End talking with a pre-built Rails API found [here](). During week 3 of this project, we'll introduce Node, Express, and Knex and you'll build a new backend using only JavaScript! In a sense, you'll be replicating the exisiting Rails API in Node (super cool).
```
Comment (tmikeschu) [inline module4/projects/quantified-self/quantified-self.md]: Super cool! I'm leaning more towards having the API built out, so they can focus on JS during the break.

Will they fork and clone that for local development, or should we plan on them all hitting the same production endpoints?

Comment (case-eee) [inline module4/projects/quantified-self/quantified-self.md]: I'm thinking they fork it 👍

File: module4/projects/quantified-self/quantified-self.md (hunk 1 above)
Comment (case-eee) [inline module4/projects/quantified-self/quantified-self.md]: haha i'll update this. agree 👍

**Comment by case-eee** [comment]
@laurenfazah i'm thinking i'll make a few updates then merge it 👍

**Pull request merged**